oxygen-account/src/main/java/com/oxygenxml/account/messages/Messages.java

```diff
@@ -9,7 +9,7 @@ public enum Messages {
 	 /**
 	  * ID corresponding to the message indicating that the username field is empty.
 	  */
-	 USERNAME_EMPTY("EMPTY_NAME", "Please insert your username."),
+	NAME_EMPTY("EMPTY_NAME", "Please insert your username."),
```

Comment:
Nu avem username, mesajul este incorect. Trebuie: "The name cannot be empty" (ce insert?)